packages/cacti-plugin-weaver-driver-fabric/package.json

```diff
@@ -1,76 +1,76 @@
 {
-    "name": "@hyperledger/cacti-plugin-weaver-driver-fabric",
-    "version": "2.1.0",
-    "description": "Driver Server for communication with a Fabric Network as part of weaver data sharing protocol",
-    "keywords": [
-        "Hyperledger",
-        "cacti",
-        "Integration",
-        "Blockchain",
-        "Distributed Ledger Technology",
-        "Weaver"
-    ],
-    "homepage": "https://github.com/hyperledger/cacti#readme",
-    "bugs": {
-        "url": "https://github.com/hyperledger-cacti/cacti/issues"
+  "name": "@hyperledger/cacti-plugin-weaver-driver-fabric",
+  "version": "2.1.0",
+  "description": "Driver Server for communication with a Fabric Network as part of weaver data sharing protocol",
```

Comment:
Can you capitalize: `weaver` --> `Weaver`?

Reply:
Sure, done the same in all package.json files that used lowercase name